config.example.toml

```diff
@@ -414,6 +414,10 @@
 # Specify the location of the Android NDK. Used when targeting Android.
 #android-ndk = "/path/to/android-ndk-r26d"
 
+# Number of parallel jobs to be used for building and testing. If set to `0` or
+# omitted, it will be automatically determined.
+#jobs = 0
```

Comment:
Worth to mention that this is used as --jobs flag on cargo.

Reply:
Added a mention.